metron-platform/metron-common/src/main/java/org/apache/metron/common/stellar/shell/StellarExecutor.java

```diff
@@ -143,11 +153,13 @@ public StellarExecutor(Console console) throws Exception {
 
   public StellarExecutor(String zookeeperUrl, Console console) throws Exception {
     this.variables = new HashMap<>();
-    this.functionResolver = new StellarFunctions().FUNCTION_RESOLVER();
     this.client = createClient(zookeeperUrl);
     this.context = createContext();
+    StellarFunctions.initialize(this.context);
+    this.functionResolver = StellarFunctions.FUNCTION_RESOLVER();
```

Comment:
I'd prefer everyone's REPL to not include the same includes and excludes as the global config, which may be used on the running cluster.  Can we, instead, have the REPL take a JSON config that can set the stellar configs?  I'm ok if we set the default config to the one in global config, but the user should be able to override.